cypress/integration/accessibility_UpsellForm_spec.js

```diff
@@ -0,0 +1,10 @@
+describe('UpsellForm Accessibility check', () => {
```

Comment:
Is this needed? It's failing the integration check and when looking through the code it looks like similar components (i.e. Table.Cell) don't have these checks

Reply:
@ayeshakmaz Do you think this can be disabled for now, or do you see any obvious issues?

Reply:
These should only be created at the page level (Upsell), so you can delete this file : ) 

It's probably something we should note when using the script to generate the files